.circleci/config.yml

```diff
@@ -525,8 +562,8 @@ jobs:
             - ~/.cache/yarn
             - node_modules
       - run:
-          name: "[FL] E2E Tests on Browserstack"
-          command: bundle exec fastlane android browserstack_e2e_tests
+          name: "[FL] Android E2E Tests on Browserstack"
+          command: bundle exec fastlane android browserstack_e2e_tests app_name:integreat-test-cms
       - run:
-          name: "[FL] E2E Tests on Browserstack"
-          command: bundle exec fastlane ios browserstack_e2e_tests
+          name: "[FL] iOS E2E Tests on Browserstack"
+          command: bundle exec fastlane ios browserstack_e2e_tests app_name:integreat-test-cms
```

Comment:
Can we split this file into multiple? This file is too large.

Reply:
I'll look into yet. I think it is possible.

Reply:
Created an issue :)